lib/script.js

```diff
@@ -0,0 +1,86 @@
+// Get the modal
+const aboutModal = document.getElementById("aboutModal");
```

Comment:
Since the idea is to use kebab-case to refer to HTTP properties, let's change them whenever they are used

```suggestion
const aboutModal = document.getElementById("about-modal");
```